OnLoop/src/CoffeeOnLoop.java

```diff
@@ -0,0 +1,162 @@
+import java.util.Scanner;
+
+public class CoffeeOnLoop {
+    static int coffeeBeans = 120;
+    static int water = 400;
+    static int milk = 540;
+    static int cups = 9;
+    static int dollar = 550;
+
+    public static void main(String[] args) {
+        String take = "take";
+        String fill = "fill";
+        String buy = "buy";
+        String exit = "exit";
+        String remaining = "remaining";
+        Scanner enter = new Scanner(System.in);
+        System.out.println("Hello! Welcome to the Coffee Machine!  PLease enter an instruction: take, fill, buy, remaining, exit");
+        String input = enter.nextLine();
+
+        while (!input.equalsIgnoreCase(exit)) {
+            if (input.equalsIgnoreCase(take)) {
+                take(dollar);
+                System.out.println("PLease enter an instruction: take, fill, buy, remaining, exit");
+                input = enter.nextLine();
+            } else if (input.equalsIgnoreCase(fill)) {
+                fill(water, milk, coffeeBeans, cups);
+                System.out.println("PLease enter an instruction: take, fill, buy, remaining, exit");
+                input = enter.nextLine();
+            } else if (input.equalsIgnoreCase(buy)) {
+                buy(dollar, water, milk, coffeeBeans, cups);
+                System.out.println("PLease enter an instruction: take, fill, buy, remaining, exit");
+                input = enter.nextLine();
+
+            } else if (input.equalsIgnoreCase(remaining)) {
+                remaining(dollar, water, milk, coffeeBeans, cups);
+                System.out.println("PLease enter an instruction: take, fill, buy, remaining, exit");
+                input = enter.nextLine();
+            }
+        }
+        System.out.println("thank you for using our services!");
+        System.exit(0);
+    }
+
+
+    public static void take(int dollar) {
+        System.out.println("I gave you " + dollar + "$");
+        CoffeeOnLoop.dollar -= dollar;
+
+    }
+
+    public static void fill(int water, int milk, int coffeeBeans, int cups) {
+        Scanner enter = new Scanner(System.in);
+        System.out.println("Write how many ml of water do you want to add:");
+        int addedWater = enter.nextInt();
+        CoffeeOnLoop.water += addedWater;
+        System.out.println("Write how many ml of milk do you want to add:");
+        int addedMilk = enter.nextInt();
+        CoffeeOnLoop.milk += addedMilk;
+        System.out.println("Write how many grams of coffee beans do you want to add");
+        int addedCoffee = enter.nextInt();
+        CoffeeOnLoop.coffeeBeans += addedCoffee;
+        System.out.println("Write how many disposable cups of coffee do you want to add: ");
+        int addedCups = enter.nextInt();
+        CoffeeOnLoop.cups += addedCups;
+
+    }
+
+    public static void buy(int dollar, int water, int milk, int coffeeBeans, int cups) {
```

Comment:
method is too big and over complicated please change it to flew small methods

Reply:
broke down method to smaller methods